### ISSUE TITLE
Size of`influx`(130+MB) and `influxd` (130+MB)

### ISSUE THREAD

**Issue by jsteenb2** [influxdata/influxdb#18866]
The latest builds of the `influx` CLI are weighing in at over 130MB. This is reallly meaty for a CLI.  These are really large binaries. 

I think it is in our best interest to understand the size and do our best to remove as much of it as possible.

Building on: `Darwin 18.7.0 x86_64`

**Comment by e-dard** [comment]
This is very likely because the current `Makefile` is building the `assets` into [both the server and CLI binaries](https://github.com/influxdata/influxdb/blob/master/Makefile#L66-L72).

If you just build the `influx` binary without the assets then it is `420KB`... The UI assets are adding about `129.5MB` to the binary build.

**Comment by jsteenb2** [comment]
seeing a bunch of interesting things using https://github.com/knz/go-binsize-viz tool to visualize the internals of the binary. We ahve about half the binary allocated to `unknown` objects, likely the assets you are mentioing @e-dard, then a lot of other goodies from all the dependencies included within the CLI.  The pictures below are helpful in understanding the breakdown of our binary size (same thing for influxd and influx mostly):

![image (7)](https://user-images.githubusercontent.com/17263167/86645506-5081c100-bf93-11ea-8e8c-4747befbde99.png)
![image (6)](https://user-images.githubusercontent.com/17263167/86645511-5081c100-bf93-11ea-8096-e00647207bc1.png)
![image (5)](https://user-images.githubusercontent.com/17263167/86645512-511a5780-bf93-11ea-956b-d182c4224957.png)
![image (3)](https://user-images.githubusercontent.com/17263167/86645513-511a5780-bf93-11ea-8e5a-869783d228c1.png)
![image (2)](https://user-images.githubusercontent.com/17263167/86645493-4d86d080-bf93-11ea-9fcf-018d266cc0d2.png)
![image (8)](https://user-images.githubusercontent.com/17263167/86645504-4eb7fd80-bf93-11ea-804c-b0b6e653c5b0.png)

**Comment by e-dard** [comment]
If we strip out symbol table and debugging we can reduce from `~130MB` to `~78MB`.

```
env GO111MODULE=on go build -tags 'assets ' -ldflags "-s -w  -X main.commit=dcc6bd197a" -o bin/darwin/influx ./cmd/influx

**Comment by shinenelson** [comment]
this issue can be closed since #19712 was merged
\- random passer-by
